src/python/dxpy/cli/dataset_utilities.py

```diff
@@ -1462,11 +1469,13 @@ def extract_assay_expression(args):
             "ending_position"
         ]["type"] = unicode
 
-    # Validate filters JSON provided by the user according to a predefined schema
-    input_json_validator = JSONValidator(
-        schema=EXTRACT_ASSAY_EXPRESSION_JSON_SCHEMA, error_handler=err_exit
-    )
-    input_json_validator.validate(input_json=user_filters_json)
+    # In case --sql flag is provided but no input json, function should return all data
+    if not return_all_data:
```

Comment:
This is a complete nitpick, however wouldn't it make a bit more sense logic wise to check it like
```suggestion
    if user_filters_json:
```